### ISSUE TITLE
How to reconstruct some unlabeled images

### ISSUE THREAD

**Issue by young169** [facebookresearch/mae#170]
the log file give me an erro like "Couldn't find any class folder in data/train."
The images are not for classification, If I give all of them one fake label, will it work? To learn some knowledge from these data?

**Comment by lennart-maack** [comment]
As far as I understand, the labels are not used during pretraining. Maybe for finetuning afterwards.
If you have a look into engine_pretrain.py line 39, they only use the samples and no labels

`for data_iter_step, (samples, _) in enumerate(metric_logger.log_every(data_loader, print_freq, header)):`
